pyproject.toml

```diff
@@ -348,6 +348,7 @@ lint.ignore = [
     "D213",  # Multi-line docstring summary should start at the second line (reason: pep257 default)
     "D215",  # Section underline is over-indented (reason: pep257 default)
     "A003",  # Class attribute shadowing built-in (reason: Class attributes don't often get bare references)
+    "A005",  # Standard library shadowing (reason: disambiguate using local names if required)
```

Comment:
```suggestion
```

This will fix the linting issue.

Reply:
But it would make ruff complain about the file. I added a per-file config ignoring this for platform.py.

Reply:
I haven't noticed that the noqa came with the master merge. Kept that and removed all my other attempts.